src/main/java/sky/Sky.java

```diff
@@ -0,0 +1,52 @@
+package sky;
+
+import sky.command.Command;
+import sky.exception.TextNoMeaningException;
+
+import java.util.Scanner;
+
+/**
+ * The Sky class is the entry point for the Sky chatbot and deals with the main logic behind its operations.
+ */
+public class Sky {
+    private TaskList taskList;
+    private Ui ui;
+    private Storage storage;
+
+    public Sky(String filePath) {
```

Comment:
Perhaps include a header comment for this constructor?

Reply:
Nice suggestion! Will add it in the future.